libursa/src/ffi/cl/verifier.rs

```diff
@@ -5,6 +5,8 @@ use ffi::ErrorCode;
 
 use std::os::raw::c_void;
 
+// TODO: Expose Verifier's add_common_attribute method
+
```

Comment:
Why add the TODO? Just implement it.

Reply:
Ok

Reply:
I have exposed for both Prover and Verifier